python/tests/server/test_worker.py

```diff
@@ -110,11 +110,15 @@
 SETUP_LOGS_FIXTURES = [
     (
         WorkerConfig("logging", setup=False),
-        ("writing to stdout at import time\n" "setting up predictor\n"),
+        (
+            "writing some stuff from C at import time\n"
```

Comment:
This changed because we now use the `StreamRedirector` for setup of normal predictors which captures libc output again. The async version uses `SimpleStreamRedirector` which does not.